SampleApp/javascript/js/SettingsTab.js

```diff
@@ -8,7 +8,9 @@ class SettingsTab extends Component {
         super(props);
         this.onLoginTapped = () => {
             console.log("onLoginTapped");
-            Iterable.setEmail(this.state.email);
+            Iterable.setEmail(this.state.email, (result) => {
+                console.log('setEmail result:'+result)
```

Comment:
So this result will have both success and failure callback data. And devs will have to basically separate it out here. Right? For typescript, perhaps this could be way to go.. 
@pauljung14 , can you advice if for front end, having one generic callback for success and failure is recommended. 

I'm thinking there is promise - resolve and reject concept. Can it apply here?

Reply:
Is the only purpose for adding this callback to log whether registering the device token failed?

Reply:
yes actually, we only need to know if it succeeded or not.

Reply:
@pauljung14 any update on this ?

Reply:
sounds fine to me